internal/vulnstore/postgres/store.go

```diff
@@ -86,83 +84,3 @@ func (s *Store) Get(ctx context.Context, records []*claircore.IndexRecord, opts
 	}
 	return vulns, nil
 }
-
```

Comment:
Winded up being too much code to just shove into the same store.go file, GC code now lives in its own file.